custom_components/ferroamp/config_flow.py

```diff
@@ -85,6 +86,10 @@ async def async_step_init(self, user_input=None):
         if precision_voltage is None:
             precision_voltage = 0
 
+        precision_frequency = self.config_entry.options.get(CONF_PRECISION_FREQUENCY)
+        if precision_frequency is None:
+            precision_frequency = 0
```

Comment:
Is zero decimals a good default value for frequency precision? Maybe we should go for 2 or more?

Reply:
I agree. Go with 2 dec as a startingpoint for frequency defaitism

Reply:
Just make the change and push a new commit. The PR will be updated automatically.